scripts/docs/en/userver/tutorial/build.md

```diff
@@ -237,7 +237,7 @@ cat scripts/docs/en/deps/arch.md | grep -oP '^makepkg\|\K.*' | while read ;\
   bash
   mkdir build_release
   cd build_release
-  cmake -DUSERVER_FEATURE_PATCH_LIBPQ=0 -DCMAKE_BUILD_TYPE=Release ..
+  cmake -DUSERVER_FEATURE_PATCH_LIBPQ=0 -DUSERVER_FEATURE_DONT_SEARCH_FOR_SYSTEM_GRPC_CMAKE=1 -DCMAKE_BUILD_TYPE=Release ..
```

Comment:
Why is it disabled for all by default?

Reply:
I will rename flag to USERVER_FEATURE_SEARCH_FOR_SYSTEM_GRPC_CMAKE. This flag will be enabled by default.